css/tabBar.css

```diff
@@ -192,32 +194,28 @@ body.dark-mode #overlay {
 /* buttons */
 
 .navbar-action-button {
-	padding: 0 1em;
+	padding: 0 0.5em;
 	height: 36px;
-	line-height: 36px !important;
+	margin-right: 0.5em;
+	margin-top: -1px;
 }
 .navbar-action-button.invisible {
 	visibility: hidden;
 }
 .navbar-action-button + .navbar-action-button {
 	padding-left: 0.5em;
 }
+.navbar-right-actions {
+	margin-left: 0.5em;
+	margin-right: 2em;
+}
 
 /* reader button */
 
 .tab-item .reader-button {
```

Comment:
The reader button seems to be misaligned now:

<img width="272" alt="Screen Shot 2020-07-16 at 11 22 35 AM" src="https://user-images.githubusercontent.com/10314059/87696639-b5d56f00-c756-11ea-9d4c-bee91824977f.png">